frontend/src/components/Loading/CircularSpinner.tsx

```diff
@@ -0,0 +1,21 @@
+import { LoadingOutlined } from '@ant-design/icons'
+
+import { Spin } from 'antd'
```

Comment:
we should replace this with our figma loading state (to remove the `@ant-design` and `antd` packages from dependencies)